examples/jku_backend.py

```diff
@@ -18,9 +18,9 @@
 def use_jku_backend():
     q_circuit = QuantumCircuit.from_qasm_file(RUNDIR  + '/ghz.qasm')
     jku_backend = JKUProvider().get_backend('local_statevector_simulator_jku')
-    result = execute(q_circuit, backend=jku_backend, shots=100).result()
+    result = execute(q_circuit, backend=jku_backend, shots=100, seed=42).result()
     print("counts: ")
     print(result.get_counts(q_circuit))
 
 if __name__ == "__main__":
-    use_jku_backend()
+    use_jku_backend()
```

Comment:
Small linter warning, while travis is not in place: can you place a newline at the end of the file?